portfolio/src/main/webapp/style.css

```diff
@@ -187,4 +197,50 @@ html, body {
     display: flex;
     justify-content: center;
     align-items: center;
+}
+
+/* TAB CONTENT */
+
+@keyframes collapseShow {
+    from {
+        height: 0;
+
+    }
+    to {
+        height: 200px;
```

Comment:
On smaller screens or bigger fonts, 200px won't be enough to show all your text. Instead, you can probably use "height: auto".

Reply:
I definitely took that into consideration when implementing this, however the animation doesn't work with "height: auto" so I'll try and find another solution.

Reply:
Oh you're right about the animation. I didn't realize that auto would break that.

In frontend web terms, this component is called an accordion. You can try to search for solutions for building an accordion. Here's a tutorial from W3C. I didn't read it closely but it might help you https://www.w3schools.com/howto/howto_js_accordion.asp

Also if it gets complicated you can always increase the height.

Reply:
I took a look and found this link which shows how to make an animated accordion with a bit of js:
https://www.w3schools.com/howto/tryit.asp?filename=tryhow_js_accordion_animate

I think this is the simplest solution I've found so far so I'll give it a shot.

Reply:
It worked! 😄